inc/editor/configuration-loader/configuration-loader.php

```diff
@@ -5,9 +5,14 @@
 defined( 'ABSPATH' ) || exit();
 
 use Error;
+use RemoteDataBlocks\Config\AirtableDatasource;
+use RemoteDataBlocks\Config\HttpDatasourceConfig;
 use RemoteDataBlocks\Config\QueryContext;
+use RemoteDataBlocks\Config\ShopifyDatasource;
 use RemoteDataBlocks\Logging\Logger;
 use RemoteDataBlocks\Logging\LoggerManager;
+use RemoteDataBlocks\REST\DatasourceCRUD;
```

Comment:
This class should probably get moved out of the `REST` namespace

Reply:
Agree.